smpp/transmitter.go

```diff
@@ -43,10 +43,11 @@ type Transmitter struct {
 	WindowSize         uint
 	r                  *rand.Rand
 
-	conn struct {
+	cl struct {
```

Comment:
because when you call e.g. (t *Transmitter) Close(), you do this in code:
t.conn.Lock()
defer t.conn.Unlock()
...
return t.conn.Close()

so one should think that closing Transmitter is closing the connection.
But no, it actually summons (c *client) Close() which actually calls c.conn.Close().

So this all is just to evade possible confusion (at least me and my colleague were confused bu conn name in Transmitter)